pkg/db/db.go

```
@@ -140,3 +167,36 @@ func InsertDataPoints(ctx context.Context, conn *pgx.Conn, elements []api.Histor
+    `, dailyAverageView)
+
+	result, err := pool.Query(ctx, query)
+	if err != nil {
+		return nil, fmt.Errorf("could not get moving averages: %w", err)
+	}
+	defer result.Close()
+
+	var averages []MovingAverages
+	for result.Next() {
+		var averagesRow MovingAverages
+		result.Scan(&averagesRow.Time, &averagesRow.Ma100, &averagesRow.Ma350x2)
```

golangci-lint (1.62.2)

Error return value of `result.Scan` is not checked

(errcheck)